scrapers/stremio_addons.py

```diff
@@ -102,9 +102,17 @@ async def process_stream(
                 stream = self.create_torrent_stream(stream_data, parsed_data, metadata)
 
                 if catalog_type == "series":
-                    if not self.process_series_data(
-                        stream, parsed_data, season, episode, stream_data
-                    ):
+                    episodes = parsed_data.get("episodes")
```

Comment:
im trusting parsed episodes of torrentio. also parsed_data.get("episodes") is None. its in `parsed_data["metadata"].get("episodes")`